lib/AbsoluteGrid.jsx

```diff
@@ -106,15 +108,15 @@ export default class AbsoluteGrid extends React.Component {
     window.removeEventListener('resize', this.onResize);
   }
 
-  onResize = () => {
+  onResize() {
```

Comment:
I much prefer the fat arrow binding style to re-assigning `this.onResize.bind` in the constructor. I think this is a step backwards.